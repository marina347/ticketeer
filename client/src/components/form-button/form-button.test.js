import React from "react";
import { shallow } from "enzyme";
import "jest-styled-components";
import FormButton from "./form-button.component";

describe("FormButton component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<FormButton />);
  });

  it("Should render FormButton component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
