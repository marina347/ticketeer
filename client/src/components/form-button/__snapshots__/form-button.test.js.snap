// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormButton component Should render FormButton component 1`] = `
<button>
  <span />
</button>
`;
